actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -480,6 +487,8 @@ def []=(name, options)
           options = { value: options }
         end
 
+        preprocess_options(options)
```

Comment:
_"We could perhaps use respond_to?(:from_now) in both cases"_

@kaspth are you suggesting to stick onto delegation? Or make the changes using ActiveSupport::Duration? :smile: